susansli/augury
# Add Buy/Sell Stocks functionality

# Description
Adds two new routes to add/sell stocks from a specific portfolio using the https://app.alpaca.markets/ API.

# Changes
- Installed `@alpacahq/alpaca-trade-api` to access real-time market data
- Created `TradeApi` singleton to get Alpaca connection
- Added `StockController` to buy and sell individual stocks
- Added `StockModel` to create/handle BuyRecords and Stock functionality
  - Has `getTotalShares` helper function
- Added basic AssertTrue/False validation functions

# Testing Prerequisites:
- Add `ALPACA_KEY_ID` and `ALPACA_SECRET` to `.env`
- Make sure you have an existing portfolio to buy stocks for
